app/javascript/geoblacklight/downloaders/downloader.js

```diff
@@ -0,0 +1,62 @@
+export default class GeoBlacklightDownloader {
+  constructor(el, options) {
+    this.el = el;
+    this.options = options || {};
+    this.options.spinner = document.createElement("div");
+    this.options.spinner.className =
+      "spinner-border spinner-border-sm float-right";
+    this.options.spinner.role = "status";
+    this.options.spinner.innerHTML = '<span class="sr-only">Downloading</span>';
+    this.options.spinner.style.display = "none";
+    // @TODO: This spinner doesn't appear to work in GBL Proper anymore.
+    //document.querySelector('.exports .card-header').appendChild(this.options.spinner);
```

Comment:
Pretty sure we can get rid of all the spinner stuff. Unless downloader is being used somewhere else, right now there is no element in the code that uses the class exports. When a download is successfully rendered it creates a flash message and there is no modal.